src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/AccessorBuilder.cs

```diff
@@ -23,12 +25,14 @@ internal static class FastInvokerBuilder
         private static readonly MethodInfo s_createSetterInternal = typeof(FastInvokerBuilder).GetMethod(nameof(CreateSetterInternal), BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Static)!;
         private static readonly MethodInfo s_make = typeof(FastInvokerBuilder).GetMethod(nameof(Make), BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Static)!;
 
+        [RequiresUnreferencedCode(DataContractJsonSerializer.SerializerTrimmerWarning)]
         public static Func<object> GetMakeNewInstanceFunc(Type type)
         {
             Func<object> make = s_make.MakeGenericMethod(type).CreateDelegate<Func<object>>();
             return make;
         }
 
+        [RequiresUnreferencedCode(DataContractJsonSerializer.SerializerTrimmerWarning)]
```

Comment:
Why - I don't see anything which should cause a problem. Linker won't see through the MakeGenericMethod because it's cached in a field, but the target method has no annotations, so it's OK to call this.